tree/tree/src/TBranch.cxx

```diff
@@ -115,6 +118,7 @@ TBranch::TBranch()
 , fEntryBuffer(0)
 , fTransientBuffer(0)
 , fBrowsables(0)
+, fBulk(new ROOT::Experimental::Internal::TBulkBranchRead(*this))
```

Comment:
humm ... this means that this is allocated for every branch no matter what.  Is that really what we want? (i.e. price paid when not using bulk I/O).